src/components/VMediaInfo/VCopyLicense.vue

```diff
@@ -0,0 +1,233 @@
+<template>
+  <div>
+    <h5 class="mb-4 text-base md:text-2xl font-semibold">
+      {{ $t('media-details.reuse.copy-license.title') }}
+    </h5>
+
+    <nav class="flex borber-b-none" role="tablist">
+      <button
```

Comment:
Just a note but this pattern isn't accessible and doesn't follow the WAI-ARIA spec for accessible tabs. I've opened this issue to create a component for it: https://github.com/WordPress/openverse-frontend/issues/749

Reply:
Thank you!